Makefile

```diff
@@ -2,7 +2,8 @@ VERSION=$(shell git describe --tags --dirty --always)
 
 .PHONY: build
 build:
-	go build -ldflags "-X 'github.com/conduitio/conduit-connector-connectorname.version=${VERSION}'" -o conduit-connector-connectorname cmd/connector/main.go
+	sed -i '/specification:/,/version:/ s/version: .*/version: '"${VERSION}"'/' connector.yaml
+	go build -o conduit-connector-connectorname cmd/connector/main.go
```

Comment:
Why not embedding the version into the binary anymore?

Reply:
That's because the source of truth will be the connector.yaml and the version needs to be taken from it.